runtime/parachains/src/configuration.rs

```diff
@@ -285,7 +285,9 @@ impl<BlockNumber: Default + From<u32>> Default for HostConfiguration<BlockNumber
 			hrmp_max_parachain_outbound_channels: Default::default(),
 			hrmp_max_parathread_outbound_channels: Default::default(),
 			hrmp_max_message_num_per_candidate: Default::default(),
-			ump_max_individual_weight: 20 * WEIGHT_PER_MILLIS,
+			ump_max_individual_weight: Weight::new()
+				.set_computation(20 * WEIGHT_PER_MILLIS)
+				.set_bandwidth(5 * 1024 * 1024),
```

Comment:
20 ms is ~1% of the 2s expected maximum. 50KB should be about the same proportion of the maximum PoV size.

```suggestion
				.set_bandwidth(50 * 1024),
```